### DISCUSSION TITLE
Normalize SOAP

### DISCUSSION THREAD

**Discussion by VondrakMar** [SINGROUP/dscribe#109] [General]
Hello,
would it be possible to add an option for getting normalized SOAP, and derivatives of normalized SOAP, please? Thank you

**Comment by lauri-codes** [comment]
Dear @VondrakMar,

Thank you for the suggestion. Are you referring to normalization with respect to Euclidean (L2) norm?

It is in our plans to offer new, more consistent set of normalization options for all descriptors, but this is not yet implemented. In the meantime, it should be possible to scale both the descriptors and derivatives by the norm with numpy.

**Comment by VondrakMar** [comment]
Dear @lauri-codes ,
yes. Scaled descriptor is easy to do, but I would need derivatives of normalized SOAP, not "normalized derivatives". This requires few lines of code of matrix multiplications, rather than simple scaling.
Thank you
